README.md

```diff
@@ -7,6 +7,34 @@ A 12 column responsive grid system for laying out documents, templates and modul
 
 [![Grid system](https://rawgit.com/Financial-Times/o-grid/master/img/grid-system.png)](https://rawgit.com/Financial-Times/o-grid/master/img/grid-system.png)
 
+## Quick Start
+
+Using the [Origami Build Service](https://build.origami.ft.com/v1/):
+
+```html
+<head>
+	…
+	<link rel="stylesheet" href="https://build.origami.ft.com/bundles/css?modules=o-grid@^3.0.0" />
+</head>
+```
+
+Or, to install it manually:
+
+```sh
+bower install o-grid --save
+```
+
+```scss
+// your-app/main.scss
+$o-grid-is-silent: false;
+@include 'o-grid/main';
```

Comment:
should this say `@import 'o-grid/main'`? Also, would it be more "origami" to show the `oGridGenerate` as a recommended way?